scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/ParseOps.scala

```diff
@@ -9,7 +9,9 @@ trait ParseOps { self: DatabaseOps =>
     def toSource: m.Source = {
       val dialect =
         m.Dialect.standards.getOrElse(language, sys.error(s"unsupported dialect $language"))
-      unit.cache.getOrElse("source", dialect(unit.source.toInput).parse[m.Source].get)
+      // TODO(olafur) remove
+      // TODO(olafur) upgrade to 2.11.12
```

Comment:
What do these todos mean?

Reply:
Removed, I wrote those down during our call and committed them for some reason 😅